addOns/bruteforce/src/main/java/org/zaproxy/zap/extension/bruteforce/OptionsBruteForcePanel.java

```diff
@@ -189,8 +192,20 @@ private JPanel getPanelPortScan() {
             gridBagConstraintsAddFileButton.insets = new Insets(2, 2, 2, 2);
             gridBagConstraintsAddFileButton.gridwidth = 1;
 
+            gridBagConstraintsBrowseFilesWithoutExtensionCheckBox.gridx = 0;
+            gridBagConstraintsBrowseFilesWithoutExtensionCheckBox.gridy = 7;
```

Comment:
Worth extracting a variable for the row to avoid changing all of them in following changes.